tests/test_phase_psf.py

```diff
@@ -200,23 +224,15 @@ def test_phase_screen_list():
 
     # Check some actual derived PSFs too, not just phase screens.  Use a small pupil_plane_size and
     # relatively large pupil_plane_scale to speed up the unit test.
-    atm.advance_by(1.0)
-    do_pickle(atm)
-    atm.reset()
-    kwargs = dict(exptime=0.06, diam=1.0, lam=1000.0)
+    atm._reset()
+    assert atm[0]._time == 0.0, "Wrong time for AtmosphericScreen"
+    kwargs = dict(exptime=0.05, time_step=0.025, diam=1.0, lam=1000.0)
```

Comment:
Sure, changed this to `kwargs = dict(exptime=0.05, time_step=0.01, diam=1.1, lam=1000.0)`